packages/tenant-process/src/routers/TenantRouter.ts

```diff
@@ -109,8 +112,8 @@ const tenantsRouter = (
           const { origin, code } = req.params;
 
           const tenant = await readModelService.getTenantByExternalId({
+            value: code,
             origin,
```

Comment:
> It's just aesthetic, but I suggest not using partial named parameters.

If we want both to be named parameters we have to disable a lint check, which i wouldn't prefer, like this:
```
            value: code,
            // eslint-disable-next-line object-shorthand
            origin: origin,
```

Otherwise we could pass those as positional parameters but I would leave like it already is because in the function that object is mapped as an `ExternalId`